app/views/videos/show.html.haml

```diff
@@ -7,6 +7,37 @@
         %header
           %h3= @video.title
           %h4=link_to @video.category.name, category_path(@video.category)
-          %span Rating: 4.5/5.0
+          %span
+            - if @video.average_rating != nil
+              = "Rating: #{@video.average_rating}/5.0"
+            - else
+              = "There are no reviews"
         %p= @video.description
         .actions
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for([@video, Review.new]) do |f|
+        = f.form_group do
+          %fieldset
+            .row
+              .col-sm-3
+                = f.select :rating, [["5 Stars", 5], ["4 Stars", 4], ["3 Stars", 3], ["2 Stars", 2], ["1 Star", 1]], { label: "Rate this video" }
+          = f.form_group do
+            .row
+              .col-sm-8
+                = f.text_area :content, label: "Write Review", :rows => 6
```

Comment:
```haml
= f.form_group do
  .row
    .col-sm-8
      = f.text_area :content, label: "Write Review", :rows => 6` 
```

should be just something like this:

```haml
= f.text_area :content, label: "Write Review", :rows => 6, control_col: "col-sm-8"
```

Since you are using bootstrap form - these Rails helpers produce already form groups rows and cols, you just have to check it exactly what is the output and study it for a bit. Play with documentation too and try to read it from top to bottom. It really helps to round up your dev skills when you work with docs.